include/miopen/config.h.in

```diff
@@ -78,11 +78,44 @@
 
 // clang-format off
 #cmakedefine MIOPEN_DEFAULT_FIND_MODE @MIOPEN_DEFAULT_FIND_MODE@
+
 // clang-format on
+#cmakedefine01 MIOPEN_OFFLINE_COMPILER_PATHS_V2
+#if MIOPEN_OFFLINE_COMPILER_PATHS_V2
+// Include rocm-core header for get ROCm Install Path Method
+#include <rocm-core/rocm_getpath.h>
+
+// Include header for PATH_MAX
+#if defined(__linux__)
+#include <linux/limits.h>
+#endif
+#ifndef PATH_MAX
+#define PATH_MAX 4096
+#endif
+
+// Max Buffer Len to hold the Path
+#define PATH_MAX_LEN (PATH_MAX + 1)
```

Comment:
We don't need PATH_MAX and PATH_MAX_LEN at all. This is wrong approach. Please use 'fs::path' instead, even for Linux.

Reply:
Thanks for comments, Updated this in latest version from https://github.com/arvindcheru/MIOpen/pull/5